### DISCUSSION TITLE
How can i disable preview in card-text content of certain post

### DISCUSSION THREAD

**Discussion by LUKEDUDE97** [cotes2020/jekyll-theme-chirpy#1360] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/115708755/0cbf8a7a-a853-46d6-9264-6dec0f4b4edc)

How can i disable this part in home page, i don't want text preview to show on some certain post. 

### Operations you have already tried

I have notice it belongs to `card-text content mt-0 mb-2`, but i don't know how to disable it.

**Comment by NichtsHsu** [comment]
Maybe you can do that by adding an option here:

https://github.com/cotes2020/jekyll-theme-chirpy/blob/5f2edb0914c0bc2254c6802d615f22cdb5b79dce/_layouts/home.html#L74-L79
